File: lab5/src/main/java/edu/miu/lab5/domain/Post.java
package com.example.lab2.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.ManyToAny;

import javax.persistence.*;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    private String title;
    private String content;
    private String author;
    @ManyToOne
    @JoinColumn(name = "id_user")
    private User user;


    @OneToMany(mappedBy = "post", cascade = CascadeType.ALL)
    private List<Comment> comments;

}

File: React/lab10/src/components/post/Post.js

import React from "react";

import Content from "../content/Content";

const Post = (props) => {

    return (
        
        <div className="Content" >
            <div> {props.id}</div>
            <div className="Field">
                {props.title}</div>
            <div className="Field">
                {props.author}
            </div>
        </div>
    
        
    );
}

export default Post;